### DISCUSSION TITLE
Welcome to TQM discussions!

### DISCUSSION THREAD

**Discussion by AustenLamacraft** [AustenLamacraft/tqm-quarto#1] [Announcements]
This is a good place to ask questions of each other about the course. I won't monitor this page closely, but may chip in from time to time if I can be of use. Note that $\LaTeX$ works, so you write mathematical formulas:

```math
\Psi_m(z_1,\ldots, z_N) = \prod_{j < k}^N (z_j-z_k)^{m} \exp\left(-\frac{1}{4}\sum_{j=1}^N\left|z_j\right|^2\right)
```

**Comment by hgothen** [comment]
Hello all,
In the chapter Lattice Models: Has anyone understood how we get from (6) to (7) ? Is this obvious or am I missing something?
Thank you and Best,
Henrik

**Comment by hgothen** [comment]
Hello again,
anyone have an Idea where the 2 comes from in 2.4 Finite Temp. Fluctuations?
```math
\frac{\partial u(x,t)}{\partial x}\Bigg|_{x=0} = c^{-1}\left(u'_\text{in}(t)-u'_\text{out}(t)\right) = c^{-1}\left(2u'_\text{in}(t)-\dot y(t)\right)
```
